pontoon/batch/utils.py

```diff
@@ -30,6 +40,29 @@ def get_translations_info(translations, locale):
     return count, translated_resources, changed_entities
 
 
+def ftl_find_and_replace(string, find, replace):
+    """Replace text values in an FTL string.
+
+    :arg string string: a serialized FTL string
+    :arg string find: a string to search for, and replace, in translations
+    :arg string replace: what to replace the original string with
+
+    :returns: a serialized FTL string
+
+    """
+
+    def replace_text_elements(node):
+        """Recursively traverse the AST and perform find and replace on text values only"""
```

Comment:
This is not recursive, it doesn't call itself. It does traverse a tree, it seems, but that's not done by this function. :)

Reply:
True :)